src/main/frontend/modules/shortcut/config.cljs

```diff
@@ -314,10 +314,18 @@
                                     :binding "mod+shift+p"
                                     :fn      (fn [] (state/toggle! :ui/command-palette-open?))}
 
-   :select-graph/open              {:desc    "Open select graph component"
-                                    :fn      (fn [] (state/set-state! :ui/open-select :select-graph))
+   :graph/open              {:desc    "Select graph to open"
+                                    :fn      (fn [] (state/set-state! :ui/open-select :graph-open))
                                     :binding "mod+shift+g"}
 
+   :graph/remove            {:desc    "Remove a graph"
+                                    :fn      (fn [] (state/set-state! :ui/open-select :graph-remove))
+                                    :binding false}
+
+   :graph/add                      {:desc "Add a graph"
+                                    :fn (fn [] (route-handler/redirect! {:to :repo-add}))
```

Comment:
For common actions like routing to a page, it would be helpful if users could define commands as data in their config e.g.
```clj
{:commands
 {:graph/add {:desc "Add a graph" :action {:id :route :args {:to :repo-add}}}}
```
I'll write up a card for this as there tradeoffs to think about for this